components/tidb_query/src/codec/datum.rs

```diff
@@ -207,6 +207,7 @@ impl Datum {
                 Ok(d.cmp(dec))
             }
             _ => {
+                // FIXME, here is not same as TiDB's
```

Comment:
It's better to keep a consistent style of `FIXME: xxx` with other snippets in this project.

Reply:
And please clarify the difference between TiDB and TiKV. (Otherwise it's a useless comment because it cannot help the reviewer to review)

Reply:
There is many difference here, so I place a FIXME here. Compare it with `Datum::compareMysqlDecimal` for detail difference